crowddata/mixins/vem.py

```diff
@@ -141,31 +142,32 @@ async def remove_at_time(self, ctx, midnight):
             return await ctx.send("Required cogs not loaded. Please alert a bot owner.")
         await dbcog.wait_until_ready()
 
-        pulls = [v for uid, v, ts in await self.config.pulls()
+        pulls = [[uid, v, ts] for uid, v, ts in await self.config.pulls()
                  if midnight <= ts < midnight + (24 * 60 * 60) and uid == ctx.author.id]
+        pulls = {char_to_emoji(str(c)): vs for c, vs in enumerate(pulls, 1)}
         if not pulls:
             return await ctx.send("There are no logged pulls during the given time period.")
 
         pulltext = []
-        for pull in pulls:
+        for c, (uid, pull, ts) in pulls.items():
             if isinstance(pull, str):
-                pulltext.append(pull)
+                pulltext.append(c + ': ' + pull)
             else:
-                pulltext.append('\t' + '\n\t'.join(
+                pulltext.append(c + '\n\t' + '\n\t'.join(
                     pdicog.monster_header.fmt_id_header(dbcog.get_monster(mid), use_emoji=True).to_markdown()
                     for mid in pull)
                                 )
         pulltext = '\n\n'.join(pulltext)
 
-        if not await get_user_confirmation(ctx, f"Are you sure you want to remove the following pulls:\n\n{pulltext}",
-                                           force_delete=False, show_feedback=True):
+        chosen = await get_user_reaction(ctx, f"Which pull would you like to remove?\n\n{pulltext}",
+                                         *pulls.keys(), NO_EMOJI, force_delete=False, show_feedback=True)
+
+        if chosen is None or chosen == NO_EMOJI:
             return
 
-        async with self.config.pulls() as pulls:
-            for pull in pulls[:]:
-                if midnight <= pull[2] < midnight + (24 * 60 * 60) and pull[0] == ctx.author.id:
-                    pulls.remove(pull)
-        await ctx.send("Your pulls have been deleted.")
+        async with self.config.pulls() as c_pulls:
+            c_pulls.remove(pulls[chosen])
+        await ctx.send("Your pull have been deleted.")
```

Comment:
Your pulls have been removed.

(missing `s`, let's consistently use "remove" as the verb)